argus/action_manager/windows.py

```diff
@@ -283,6 +286,50 @@ def is_dir(self, path):
         """
         return self._exists(path, self.PATH_CONTAINER)
 
+    def _new_item(self, path, item_type):
+        """Create a New-Item.
+
+        :param path:
+            Instance path of the new item.
+        :param item_type:
+            It can be `Directory` or `File`
+        """
+        cmd = "New-Item -Path {} -Type {} -Force".format(path, item_type)
```

Comment:
The path should be enclosed within quotation marks.
Also, you can use named groups.